data_fusion/plugins/yaml/products/stitched.yaml

```diff
@@ -22,7 +22,7 @@ spec:
             arguments: {}
         colormapper:
           plugin:
-            name: linear_norm
+            name: matplotlib_linear_norm
```

Comment:
@evrose54  we need to make sure this gets updated back to linear_norm once the geoips PR has been merged.

Reply:
Sounds good. Once [NRLMMD/geoips#504](https://github.com/NRLMMD-GEOIPS/geoips/pull/504) is merged we can make the change back to linear norm.